apps/web/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { DropdownOptions } from './generic/DropdownOptions';
+import { planningFormDropdown } from '../styles/styles';
+
+interface DropdownOptionProps {
+  id: string;
+  value: string;
+}
+interface DropdownProps {
+  dropdownOptions: DropdownOptionProps[];
+}
+
+export const Dropdown = ({ dropdownOptions }: DropdownProps) => {
```

Comment:
I don't think this file will be needed. The code can be copied to CareLocationSelector